src/common/kernel.cpp

```diff
@@ -46,7 +46,8 @@
 #endif
 #endif
 
-int    runflag = 1;
+std::atomic<bool> runFlag = true;
```

Comment:
Maybe rename this gRunFlag? Since it's mega-global

Reply:
Will do 👍